### ISSUE TITLE
FormAction.doValidate() isn't called when there is no validator configured [SWF-397]

### ISSUE THREAD

**Issue by spring-operator** [spring-projects/spring-webflow#195]
**[Andreas Schildbach](https://jira.spring.io/secure/ViewProfile.jspa?name=goonie)** opened **[SWF-397](https://jira.spring.io/browse/SWF-397?redirect=false)** and commented

FormAction.doValidate() isn't called when there is no validator configured.

This is no problem for the default implementation of FormAction.doValidate(), which would only invoke configured validators.

In the case of a custom doValidate() implementation (which is encouraged in the JavaDoc of validate()) it is very likely that you won't configure a Validator object. Neverless, you'd like your custom validation to be executed.

In other words, not bindAndValidate() should decide wether to call any Validators (strip "if (getValidator() != null"), but the default implementation of doValidate(). I can provide a patch if you'd like.

---

**Affects:** 1.0.3
